extensions/2.0/Khronos/KHR_xmp/README.md

```diff
@@ -0,0 +1,337 @@
+| [ProperName](https://github.com/adobe/xmp-docs/blob/master/XMPNamespaces/XMPDataTypes/CoreProperties.md#propername)        | String formatted as described in the XMP documentation |
+| [RenditionClass](https://github.com/adobe/xmp-docs/blob/master/XMPNamespaces/XMPDataTypes/CoreProperties.md#renditionclass)    | String formatted as described in the XMP documentation |
+| [URI](https://github.com/adobe/xmp-docs/blob/master/XMPNamespaces/XMPDataTypes/CoreProperties.md#uri)                      | String formatted as described in the XMP documentation |
+| [ResourceRef](https://github.com/adobe/xmp-docs/blob/master/XMPNamespaces/XMPDataTypes/CoreProperties.md#resourceref)      | Object, we refer to the XMP documentation for a description of its properties |
+| [URL](https://github.com/adobe/xmp-docs/blob/master/XMPNamespaces/XMPDataTypes/CoreProperties.md#url)                      | String formatted as described in the XMP documentation |
+| [Rational](https://github.com/adobe/xmp-docs/blob/master/XMPNamespaces/XMPDataTypes/CoreProperties.md#rational)            | String formatted as described in the XMP documentation |
+| [FrameRate](https://github.com/adobe/xmp-docs/blob/master/XMPNamespaces/XMPDataTypes/CoreProperties.md#framerate)          | String formatted as described in the XMP documentation |
+| [FrameCount](https://github.com/adobe/xmp-docs/blob/master/XMPNamespaces/XMPDataTypes/CoreProperties.md#framecount)        | String formatted as described in the XMP documentation |
+| [Part](https://github.com/adobe/xmp-docs/blob/master/XMPNamespaces/XMPDataTypes/CoreProperties.md#part)                    | String formatted as described in the XMP documentation |
+
+### Additional Properties
+
+| XMP Property Type | JSON Type |
+| ----------------- | --------- |
+| [ResourceEvent](https://github.com/adobe/xmp-docs/blob/master/XMPNamespaces/XMPDataTypes/ResourceEvent.md)| Object, we refer to the XMP documentation for a description of its properties |
+
+## Defining XMP Metadata
+
+An indirection level is introduced to avoid replicating the same XMP metadata in multiple glTF nodes.
+XMP metadatas are defined within a dictionary property in the glTF scene description file by adding an `extensions` property to the top-level glTF 2.0 object and defining a `KHR_xmp` object. The `KHR_xmp` object defines two properties:
+
+* `namespaces` : a dictionary mapping XMP [namespaces](https://github.com/adobe/xmp-docs/blob/master/Namespaces.md) to the URI where they are defined.
+* `packets`: an array of metadata packets referencing the namespaces defined in the `namespaces` dictionary.
+
+The following example defines a glTF scene with a sample XMP metadata.
+
+```json
+"extensions": {
+    "KHR_xmp" : {
+        "namespaces" : {
+            "dc" : "http://purl.org/dc/elements/1.1/"
```

Comment:
Here is what I wrote to @emilian0 about this:
>The answer depends on whether the person is processing as JSON or JSON-LD.  If they are doing only JSON processing, then they can just look for dc:title and ignore the actual values of in @context.  However, if they are doing JSON-LD processing, then they will use the @context accordingly.   And again, this is just JSON/JSON-LD - nothing specific to XMP.

He pointed out that this PR is currently using `namespaces` instead of a proper `@context` to comply with ISO 16684-3.  So that needs to change as well to make the above more correct.

Reply:
yes. Following Leonard recommendation I will make a pull request to move from our custom namespace mechanism to JSON-LD (that is used in ISO 16684-3). And yes, as a consequence, to correctly process the JSON-LD you will have to interpret the `@context` attribute.

Reply:
I'm not immediately managing to find a copy of `ISO 16684-3`, is that available somewhere? 

And, is there a reason for the `@` in `@context`? This would be the only occurrence of that prefix in the glTF specification.

Reply:
No, 16684-3 is still in development (DIS stage, in ISO speak).  

`@context` is required because the serialization is actually JSON-LD and not plain JSON.  JSON-LD requires the use of `@context` to define the namespaces used - see https://w3c.github.io/json-ld-syntax/#the-context

Reply:
I see, thanks. Should this specification mention JSON-LD somewhere? Will this information be useful to implementors of the extension?

Reply:
I'm frankly not familiar with JSON-LD or XMP, and I expect most implementors will not be either. If this is a significant motivation for the serialization/deserialization design I believe the topic should be given some explanation in the extension.

Reply:
Yes, JSON-LD should indeed be mentioned in the docs.  Good point...

Reply:
Bumping this back up — could JSON-LD be mentioned in this extension's documentation?